spec/openapi.yaml

```diff
@@ -4058,7 +4058,7 @@ paths:
           {
             "key_id": "7ca628a8-08b0-4e42-aeb8-960b37049c31",
             "cipher": "RSA",
-            "digest": "SHA224",
+            "digest": "SHA256",
```

Comment:
Recommend HMAC SHA256 for authentication only, not in case of Signature Key Management

Reply:
Okay! will update